### DISCUSSION TITLE
New CFWheels user

### DISCUSSION THREAD

**Discussion by cfcoder** [wheels-dev/wheels#1298] [General]
Brand new to CFWheels! Just installed the server and app using CommandBox. So far the documentation has been great!

Just wanted to point out a stale link. In the default home page for my new generated app (after starting the server with the Adobe2018 engine) I clicked on the Hello World tutorial link (https://guides.cfwheels.org/v2.4/docs/beginner-tutorial-hello-world) and it doesn't work. Not a big deal, but maybe someone knows how to update that for future users getting started. :)

Looking forward to learning this new framework!

**Comment by timsayshey** [comment]
![image](https://media.giphy.com/media/NHh7D7qR0LTSDtfu8p/giphy-downsized.gif)

Welcome aboard! Great choice, I've been a happy wheels dev for over a decade now and built a successful startup on it.

**Comment by bpamiri** [comment]
Welcome. Keep dropping your comments in here. It’s good to have a new pair of eyes looking at the docs and stuff and pointing out stuff like this.

**Comment by MvdO79** [comment]
Hi welcome! Thanks for keeping an eye. It's a great framework with a lot of documentation en devs that are willing to help. Tip: the old CfWheels Google group can be used as an extensive archive.

**Comment by cfcoder** [comment]
Thanks! I appreciate the tip on the archives. I’m sure I’ll need to reference that for help.

Bruce

**Comment by cfcoder** [comment]
Just found a problem in the database tutorial
https://guides.cfwheels.org/cfwheels-guides/introduction/readme/beginner-tutorial-hello-database

The callout box has you download the database SQL from github. The SQL creates a users table with a column "username". But the tutorial refers to the column and field name in the new user form as "name".

Another discrepancy is the SQL includes the creation of "passwd" as the col name but in the tutorial it is referred to as "password".

**Comment by cfcoder** [comment]
Here's another typo:

![image](https://github.com/cfwheels/cfwheels/assets/4595721/04e1578e-f15e-47b3-8f8c-d7639befd464)

**Comment by MvdO79** [comment]
Thanks @cfcoder for your finds. For others: Is there any way we could get access to GitBook to make changes or should this kind of documentation errors be reported by referencing it in a new issue?

**Reply by bpamiri** [reply]
The GitBooks guides are driven off the `guides` directory in this repository. Simply submit a PR and once merged it will automatically update the GitBook site.

**Reply by MvdO79** [reply]
Thanks, will do!

**Comment by bpamiri** [comment]
Thank you @MvdO79 for the two PRs #1304 and #1305. These have been merged and the guides have been updated.

**Comment by cfcoder** [comment]
Thanks for the updates.
I'm on the 3rd tutorial for AJAX.  As feedback from fresh eyes, I think it would be helpful to explain what the route "to" attribute "say##hello" means to CFW.  I see from debugger mode that it creates an action "say-hello" with the query params. How does CFW interpret that? No explanation about it. I didn't have the functions added in the Say.cfc yet and nothing was working. I tried adding a "pattern='say/hello'" but that didn't do anything. I then added the Say.cfc additions and still didn't work till I removed that pattern. Then it worked. But how is it working with say-hello action?

**Reply by MvdO79** [reply]
Thanks again, created an issue and will check it!

**Comment by cfcoder** [comment]
Also found a dead link in that tutorial
![image](https://github.com/cfwheels/cfwheels/assets/4595721/a84400c2-7ada-490d-9838-c453a37f3848)
